### PR TITLE
Task 4.1

### PR DESCRIPTION
**Pull request by akravchenko564**

### CODE REVIEW

File: Practicum/JSBook/Z4.1.js [hunk 1]
```diff
@@ -0,0 +1,5 @@
+﻿var person = {};
+person.name = 'Max';
+person.surname = 'Yiii';
+person.name = 'Masim'
+delete person.name;
```
Comment (KasichMax) [inline Practicum/JSBook/Z4.1.js]: How to verify that this code is working?
May be add some logs? For example:
```
console.log("Name before delete = " + person.name);
delete person.name;
console.log("Name after delete = " + person.name);
```

File: Practicum/JSBook/Z4.1.html [hunk 2]
```diff
@@ -0,0 +1,10 @@
+﻿<!DOCTYPE html>
+<html>
+<head>
+	<title>ex3</title>
+	<meta charset="utf-8" />
+	<script src="ex3.js"></script>
```
Comment (KasichMax) [inline Practicum/JSBook/Z4.1.html]: Why ex3.js?
Should be Z4.1.js.

File: Practicum/JSBook/Z4.1.html [hunk 3]
```diff
@@ -0,0 +1,10 @@
+﻿<!DOCTYPE html>
+<html>
+<head>
+	<title>ex3</title>
```
Comment (KasichMax) [inline Practicum/JSBook/Z4.1.html]: Z4.1?

File: Practicum/JSBook/Z4.1.js (hunk 1 above)
Comment (akravchenko564) [inline Practicum/JSBook/Z4.1.js]: Done c5f4f4c0fa0404bcbcdb03a85e824efa4e059215

File: Practicum/JSBook/Z4.1.html (hunk 2 above)
Comment (akravchenko564) [inline Practicum/JSBook/Z4.1.html]: Done c5f4f4c0fa0404bcbcdb03a85e824efa4e059215

File: Practicum/JSBook/Z4.1.html (hunk 3 above)
Comment (akravchenko564) [inline Practicum/JSBook/Z4.1.html]: Done c5f4f4c0fa0404bcbcdb03a85e824efa4e059215

**Pull request merged**